index.html

```diff
@@ -0,0 +1,1224 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NYT</title>
+    <link rel="stylesheet" href="css/style-storybody.css">
+    <link rel="stylesheet" href="css/styles-mainHeader.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/styles-headerArticle.css">
+    <link rel="stylesheet" href="css/styles-lowerFooter.css">
+    <link rel="stylesheet" href="css/styles-bottomNav.css">
+    <link rel="stylesheet" href="css/styles-recirc.css">
```

Comment:
`href="css/styles-recirc.css"`  is not a correct path please use like `href="./css/styles-recirc.css"` (**update all links**)